torch/csrc/autograd/functions/batch_normalization.cpp

```diff
@@ -17,6 +17,10 @@ namespace torch { namespace autograd {
 
 using thpp::Tensor;
 
+#ifndef CUDNN_BN_MIN_EPSILON
+#define CUDNN_BN_MIN_EPSILON 0
```

Comment:
if pytorch is not compiled with CUDNN, then this variable wont be defined. If this wont be defined, we can define it as 0, because use_cudnn will be False and we will be using THNN anyways.